content/04.05.00.results_framework.md

```diff
@@ -3,25 +3,28 @@
 ![
 **Schematic of the PhenoPLIER framework.**
 **a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (single-variant).
-PhenoPLIER integrates groups of genes co-expressed in specific cell types (gene modules) with gene-trait and gene-drug associations.
-At the bottom-right corner, we show how a drug downregulates two genes (A and C) that are part of the disease-associated Gene module 1.
-**b)** The integration consists of projecting gene-trait/gene-drug associations from PhenomeXcan/LINCS L1000 (bottom) to a latent space based on gene modules (represented by latent variables/LVs) from MultiPLIER (top).
-This process generates matrices $\mathbf{\hat{L}}$ and $\mathbf{\hat{M}}$, where LVs now describe each drug/trait.
-In the middle, we show the three main computational components provided by PhenoPLIER to perform this integration:
-1) an LV-based regression model,
-2) a clustering framework to learn groups of traits,
-and 3) an LV-based drug repurposing approach.
-**c)** LV603, termed as a neutrophil signature in the original MultiPLIER study, was associated in PhenoPLIER with neutrophil counts and other white blood cells (bottom, showing the top 10 traits for LV603 after projecting gene-trait associations in PhenomeXcan).
-Genes that are part of LV603 were expressed in relevant cell types (top).
+In GWAS, we identify variants associated with traits.
+In TWAS, first, we identify variants that are associated with gene expression levels (eQTLs); then, prediction models based on eQTLs are used to impute gene expression, which is used to compute gene-trait associations.
+Resources such as LINCS L1000 provide information about how a drug perturbs gene expression; at the bottom-right corner, we show how a drug downregulates two genes (A and C).
+In PhenoPLIER, all these data types are integrated using groups of genes co-expressed in specific conditions (such as cell types) that we call gene modules or latent variables/LVs.
```

Comment:
```suggestion
In PhenoPLIER, these data types are integrated using groups of genes co-expressed across one or more unlabeled conditions (such as cell types) that we call gene modules or latent variables/LVs.
```
Trying to get the idea in here that the conditions exist but are unknown - I almost said latent conditions, but I think unlabeled -> latent variables might get that connection as well...